packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -52,6 +53,10 @@ export const ParticipantListWithHeading = (props: {
 }): JSX.Element => {
   const { onFetchAvatarPersonaData, onFetchParticipantMenuItems, title, participantListProps } = props;
   const theme = useTheme();
+  /* @conditional-compile-remove(total-participant-count) */
+  const adapter = useAdapter();
+  /* @conditional-compile-remove(total-participant-count) */
+  const totalParticipantCount = adapter.getState().call?.totalParticipantCount;
```

Comment:
maybe there is a way to make this one prop that we get through the selector and we light it up for the participantList selector? totalParticipantCount

Reply:
yup totally forgot the props are valid here. will change this!